editor/edit-post/header/plugins/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getSidebars, activateSidebar } from '../../../api/sidebar';
+import { MenuItemsGroup } from '../../../../components';
```

Comment:
We should not be reaching into other top-level directories. This should be `@wordpress/components` (and moved under "WordPress dependencies" comment)